tests/semanticdb/expect/StructuralTypes.expect.scala

```diff
@@ -16,7 +16,7 @@ object StructuralTypes/*<-example::StructuralTypes.*/:
 
   val V/*<-example::StructuralTypes.V.*/: Object/*->java::lang::Object#*/ {
     def scalameta/*<-local4*/: String/*->scala::Predef.String#*/
-  } = /*<-local6*/new:
-    def scalameta/*<-local5*/ = "4.0"
+  } = new:
+    /*<-local6*/def scalameta/*<-local5*/ = "4.0"
```

Comment:
I just run `scala3-compiler/Test/runMain dotty.tools.dotc.semanticdb.updateExpect` and, to be honest, I'm not sure what this change means.

> Expect files are used as regression tests to detect changes in the compiler.
> __Their correctness is determined by human inspection__


Which is correct, the current one or the former one...?:thinking:

Reply:
If `<-local6` is supposed to point to local anonymous class, I feel the current one is correct.